src/utils/index.js

```diff
@@ -0,0 +1,47 @@
+const parseCarName = names => {
```

Comment:
바로 return을 하는 함수라면 ()안에 return을 내포하고 있어 아래와 같이 리펙토링이 가능합니다!
```suggestion
const parseCarName = names => (
  names.split(',').map(name => name.trim())
)
// 혹은 
const parseCarName = names => names.split(',').map(name => name.trim())
```